### ISSUE TITLE
How many epoch you trained?

### ISSUE THREAD

**Issue by gsygsy96** [batsa003/videogan#3]
Hello, Can you tell me how many epochs you used?

**Comment by batsa003** [comment]
I was able to run 2 epochs. The training set is very large, so even 1 epoch takes a long time.

**Comment by gsygsy96** [comment]
I find some bug in your code.
1.RGB channels is not matched cause you use opencv to read your image.
2.In dataloader, I can't understand why you shuffled training images. Isn't is will break the video's sequence?
